src/app/ReadClient.h

```diff
@@ -118,6 +118,7 @@ class ReadClient : public Messaging::ExchangeDelegate
 
         /**
          * OnSubscriptionEstablished will be called when a subscription is established for the given subscription transaction.
+         * If using auto resubscription, OnSubscriptionEstablished will be called whenever resubscription is established
```

Comment:
```suggestion
         * If using auto resubscription, OnSubscriptionEstablished will be called whenever resubscription is established.
```

This was marked resolved, but not done...